Modules not found

Hello,
I just did a fresh install and installed the library right away, meaning, there is no other library installed and nothing could affect this library in a negative way.
I get the following error when compiling though:

`ERROR in ./node_modules/twisted/dist/base/base.js
Module not found: Error: Can't resolve 'dotenv' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\twisted\dist\base'
ERROR in ./node_modules/request/lib/har.js
Module not found: Error: Can't resolve 'fs' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\request\lib'
ERROR in ./node_modules/forever-agent/index.js
Module not found: Error: Can't resolve 'net' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\forever-agent'
ERROR in ./node_modules/tough-cookie/lib/cookie.js
Module not found: Error: Can't resolve 'net' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\tough-cookie\lib'
ERROR in ./node_modules/tunnel-agent/index.js
Module not found: Error: Can't resolve 'net' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\tunnel-agent'
ERROR in ./node_modules/forever-agent/index.js
Module not found: Error: Can't resolve 'tls' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\forever-agent'
ERROR in ./node_modules/tunnel-agent/index.js
Module not found: Error: Can't resolve 'tls' in 'C:\Users\justi\WebstormProjects\lolweb\node_modules\tunnel-agent'
`

It makes it impossible to compile unfortunately.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "twisted",
-  "version": "1.17.0",
+  "version": "1.18.0",
   "description": "Fetching riot games api data",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",

--- a/package.json
+++ b/package.json
@@ -53,7 +53,6 @@
     "@typescript-eslint/eslint-plugin": "^2.1.0",
     "@typescript-eslint/parser": "^2.1.0",
     "chai": "^4.2.0",
-    "dotenv": "^8.1.0",
     "mocha": "^6.2.0",
     "sinon": "^7.4.2",
     "ts-node": "^8.3.0",
@@ -63,6 +62,7 @@
   },
   "dependencies": {
     "camelcasejs": "^1.0.3",
+    "dotenv": "^8.1.0",
     "http-status-codes": "^1.3.2",
     "lodash": "^4.17.15",
     "promise-queue": "^2.2.5",

